nucypher/characters/lawful.py

```diff
@@ -390,12 +386,12 @@ class Bob(Character):
 
     _default_crypto_powerups = [SigningPower, DecryptingPower]
 
-    class IncorrectCFragReceived(Exception):
+    class IncorrectCFragsReceived(Exception):
         """
-        Raised when Bob detects an incorrect CFrag returned by some Ursula
+        Raised when Bob detects incorrect CFrags returned by some Ursulas
         """
-        def __init__(self, evidence):
-            self.evidence = evidence
+        def __init__(self, grievances):
+            self.grievances = grievances
```

Comment:
I noticed this in my first pass through, but it's not really sitting well with me.

Reply:
Yeah, I also had second thoughts with these two lines in particular. I'll revert it.

Reply:
Wow - This is Grim

Reply:
✅